provider/cmd/pulumi-resource-aws-static-website/website.ts

```diff
@@ -129,9 +129,20 @@ export class Website extends pulumi.ComponentResource {
     }
 
     private getOrganizationName(): string {
-        const [ organization ] = execSync(`pulumi --stack='${pulumi.getStack()}' stack`)
+        const config = new pulumi.Config();
+        const organizationNameOverride = config.get("pulumiOrganizationName");
+        if (organizationNameOverride && organizationNameOverride != "") {
```

Comment:
I think this can be simplified to just `if(oranizationNameOverride)`, since I believe empty string is a falsey value anyway.

Reply:
Yeah haha, I keep thinking an empty string is true in node 🤦